### PR TITLE
Add Airflow example DAGs of tensorflow and kmeans

### PR DESCRIPTION
**Pull request by yangaws**
*Issue #, if available:*

*Description of changes:*

Add 3 DAG examples of using SageMaker in Airflow:
1. KMeans low level training example
2. TF training and transform in Python Operator
3. TF training and transform in SageMaker Operator

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by laurenyu** [comment]
don't forget the license header. Also, does it make sense to put examples in the SDK like this?

File: src/sagemaker/workflow/examples/scripts/tf_mnist.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import tensorflow as tf
+from tensorflow.python.estimator.model_fn import ModeKeys as Modes
+
+INPUT_TENSOR_NAME = 'inputs'
+SIGNATURE_NAME = 'predictions'
+
+LEARNING_RATE = 0.001
+
+
+def model_fn(features, labels, mode, params):
```
Comment (laurenyu) [inline src/sagemaker/workflow/examples/scripts/tf_mnist.py]: is this legacy mode?

File: src/sagemaker/workflow/examples/kmeans_training_lowlevel.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    'retry_delay': timedelta(minutes=1)
+}
+
+dag = DAG('kmeans_training_lowlevel', default_args=default_args,
+          schedule_interval='@once')
+
+# Constants
+role = 'my_sagemaker_role'
+job_name = 'kmeans-lowlevel-' + strftime("%Y-%m-%d-%H-%M-%S", gmtime())
+
```
Comment (laurenyu) [inline src/sagemaker/workflow/examples/kmeans_training_lowlevel.py]: maybe add the dataset location to the list of constants?

File: src/sagemaker/workflow/examples/kmeans_training_lowlevel.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+def prepare_resources(**context):
+    bucket = Session().default_bucket()
+
+    # Load the input dataset
+    request.urlretrieve("http://deeplearning.net/data/mnist/mnist.pkl.gz", "mnist.pkl.gz")
+    with gzip.open('mnist.pkl.gz', 'rb') as f:
```
Comment (laurenyu) [inline src/sagemaker/workflow/examples/kmeans_training_lowlevel.py]: `'mnist.pkl.gz'` should be a variable

**Comment by yangaws** [comment]
Still need to think whether to put these examples in python SDK. Close this PR for now.

**Pull request closed without merging**